phpunit.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- TODO set checkForUnintentionallyCoveredCode="true" once https://www.drupal.org/node/2626832 is resolved. -->
+<!-- PHPUnit expects functional tests to be run with either a privileged user
+ or your current system user. See core/tests/README.md and
+ https://www.drupal.org/node/2116263 for details.
+-->
+<phpunit bootstrap="/Applications/MAMP/htdocs/ezcontent/docroot/core/tests/bootstrap.php" colors="true"
```

Comment:
@arshadkhan53 Avoid using the hardcoded path for the bootstrap value. Can you please test if this works with **core/tests/bootstrap.php** instead of specifying the absolute path.

Reply:
Hi @riteshgurung ,
Thanks for your comment, phpunit.xml file is  template version which user can customize as per their environment, we don't even need this its there in core , i would be removing this from pr.